nf_core/pipeline-template/.github/workflows/ci.yml

```
@@ -11,6 +11,10 @@ on:
 env:
   NXF_ANSI_LOG: false
 
+concurrency: 
+  group: ${{ github.ref }}
```

```suggestion
group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```
That's how we have it for nf-core/tools

---

This adds a bit more context to the job name so that its easier later to track down what was tested.

---

awesome thanks for the explanation